src/corona_hakab_model/generation/matrix_generator.py

```diff
@@ -28,13 +31,23 @@ def __init__(self):
         self.matrix = None
 
     # todo make this work, using the parasymbolic matrix serialization.
-    def export(self, export_path, file_name: str):
-        pass
+    def export(self, export_path: str):
+        with open(export_path+'.'+self.matrix_type, 'wb') as f:
+            bsa.universal.write(self.matrix, f)
```

Comment:
have you tested this to make sure that it works on the parasymbolic matrix?

Reply:
Yes, I've checked it on the parasymbolic matrix type.